### ISSUE TITLE
Implement function "add music from local files" to maeve

### ISSUE THREAD

**Issue by ihusky02** [antrancs/maeve#20]
Good Mornng, I want a know can you add this function (similar to Google Music). I don`t want back again to this Windows 10 tragedy to add music into itunes :(

**Comment by antrancs** [comment]
Hi, as I just replied in the other thread, this is a nice suggestion but it requires making a separate desktop version for Maeve. I'll consider it.

But quick question about Google Play Music, I haven't used it before but I think you have to upload your local files to their server and play them right?

**Comment by ihusky02** [comment]
> Hi, as I just replied in the other thread, this is a nice suggestion but it requires making a separate desktop version for Maeve. I'll consider it.
> 
> But quick question about Google Play Music, I haven't used it before but I think you have to upload your local files to their server and play them right?

Yes, I use Google Music everyday, and I can move file to open Google Music in Chrome and sync with Google Music, but it require a small desktop app

**Comment by antrancs** [comment]
I see. You have to upload your songs to their server anyway. There's no official API to upload your songs to iCloud Music Library right now. You have to do that through iTunes.
